vocode/streaming/synthesizer/base_synthesizer.py

```diff
@@ -242,6 +244,52 @@ async def chunk_generator(output_bytes):
             ),
         )
 
+    async def mp3_streaming_output_generator(
```

Comment:
Nit: This is mostly a repeat of experimental_mp3_streaming_output_generator, could probably be unified